src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  const midSale = 20;
+  const bigSale = 50;
+  const lowDayPrice = days * cost;
+  const midDayPrice = lowDayPrice - midSale;
+  const bigDayPrice = lowDayPrice - bigSale;
+
+  if (days >= 7) {
+    return bigDayPrice;
+  }
```

Comment:
make calculation only if you need it
```suggestion
  if (days >= 7) {
    return lowDayPrice - bigSale;
  }
```